setup.py

```diff
@@ -42,16 +42,44 @@
     exec(f.read(), {}, version_ns)
 
 
+long_desc = """
```

Comment:
Can we just read in the `README.md` and use that as the `long_desc`?

```python
with open(os.path.join(here, "README.md")) as f:
    long_description = f.read()

long_description = long_description
long_description_content_type = "text/markdown"
```

Reply:
We could, but there are some github specific thing in the readme. Not sure it matters much.